templates/header.hbs

```diff
@@ -34,15 +34,18 @@
       </div>
 
       <div class="header__menu__socials">
-          <a class="link" href="http://www.facebook.com">
+        {{#each socialLinks}}
+        {{{module this}}}
+        {{/each}}
+          {{!-- <a class="link" href="http://www.facebook.com">
```

Comment:
Pleasae remove this commented code

Reply:
.) deleted 228f9c6bb588e8e43122e8a409535ef3b9a080d7